assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.stories.js

```diff
@@ -62,6 +62,14 @@ NotificationCTA.args = {
 		registry.dispatch( CORE_SITE ).receiveGetNotifications( [ notification1 ], {} );
 	},
 };
+NotificationCTA.scenario = 	{
+	kind: 'Global',
+	name: 'DashboardCoreSiteAlerts1',
+	story: 'DashboardCoreSiteAlerts1',
+	hierarchyRootSeparator: '|',
+	hierarchySeparator: {},
+	readySelector: '.googlesitekit-publisher-win',
```

Comment:
Ideally these three would be nested inside `options` (maybe inside `parameters`) but currently `@babel/traverse` is returning `undefined` for any nested objects (I haven't looked at this at all though!)

Reply:
This object should only include properties of a [Backstop scenario](https://github.com/garris/BackstopJS#advanced-scenarios). So these three are actually the only properties that belong here 😄 

The Story `id` and `kind` should all be same based on the existing definition so we shouldn't need anything special for that here.

Reply:
Changed now. All VRTs passing :+1: